code/src/CharachteristicsButtonOption.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const CharacteristicsButtonOptions = ({
+    characteristicsName, 
+    setInputCharacteristics, 
+}) => {
+
+const onInputCharacteristicsChanged = (e) => {
+    setInputCharacteristics(e.target.value);
+    console.log(e.target.value)
```

Comment:
In order to keep code clean, make sure to remove console.log before publishing :)